.github/steps/1-step.md

```diff
@@ -36,11 +36,14 @@ Let's start up our project environment. This lab will be ran entirely from withi
 
 Now let's configure [Azure DevOps](https://dev.azure.com/) credentials so we can create a mock project for this lab.
 
+> [!NOTE]
+> If you do not have an Azure DevOps account you can sign up for free by following this [guide](https://learn.microsoft.com/azure/devops/user-guide/sign-up-invite-teammates?view=azure-devops&tabs=microsoft-account). Once you have an account you can create a new organization using these [instructions](https://learn.microsoft.com/azure/devops/organizations/accounts/create-organization?view=azure-devops).
+
 1. Navigate to your [Azure DevOps](https://dev.azure.com/) organization in a new browser tab.
 1. Click on `User Settings` in top right corner, next to your profile picture.
 1. Select `Personal access tokens` from the dropdown menu. Click `New Token` to create a new personal access token.
 1. Give your token a descriptive name (e.g., `GitHub Skills Migration Exercise`) and set the expiration to a shorter duration (e.g., 1 day).
-1. Under `Scopes`, select `Custom defined` and choose the following scopes:
+1. Under `Scopes`, select `Custom defined` and choose the following scopes ([reference](https://docs.github.com/migrations/using-github-enterprise-importer/migrating-from-azure-devops-to-github-enterprise-cloud/managing-access-for-a-migration-from-azure-devops#personal-access-tokens-for-azure-devops)):
```

Comment:
I'll leave it out to avoid confusion that the lesser scoped token could work in this exercise.
```suggestion
1. Under `Scopes`, select `Custom defined` and choose the following scopes:
```